### ISSUE TITLE
Swift 2.1 build

### ISSUE THREAD

**Issue by ky1ejs** [nickoneill/PermissionScope#110]
Hey Nick, the current .zip will not work with Xcode 7.1 and Swift 2.1. Could you build, archive and upload a new version using Xcode 7.1 please? :smile: thanks!

**Comment by nickoneill** [comment]
Hey @kylejm, I have a few things to include in the next release, this'll definitely be one of them.

**Comment by ky1ejs** [comment]
Thanks @nickoneill!

For anyone else having this problem, you can get around it by using:

``` bash
carthage bootstrap --no-use-binaries
```

to build Permission Scope with Xcode 7.1.

**Comment by ky1ejs** [comment]
@nickoneill @bre7 it's been a long time since last release. Can one be expected soon? :)

**Comment by bre7** [comment]
If I'm not mistaken, 1.0.2 will include some bug fixes: #125, #127...[among others](https://github.com/nickoneill/PermissionScope/milestones/1.0.2)) I haven't had the time to check them out lately.

**Comment by ky1ejs** [comment]
Awesome, thanks @bre7! Any idea when we can expect 1.0.2? The bug fixed by #114 would be particularly nice to have. I was slightly confused by this, since I thought I'd fixed it with PR #100.

**Comment by nickoneill** [comment]
Hey @kylejm, we'll get 1.0.2 out the door this week. Sorry for the wait.

**Comment by ky1ejs** [comment]
Sweet! Legend!

**Comment by tmspzz** [comment]
@nickoneill any estimate on this? 

also I'm getting this when building with carthage

```
*** WARNING: Skipping build for "iphoneos" SDK because the necessary signing identity "iPhone Distribution" is not installed
```

Thanks for the great work!

**Comment by ky1ejs** [comment]
@blender this is because you don't have a distribution signing cert and key on your machine.

@nickoneill I think we should change the signing identity back to be `iOS Developer` just in case any developers that don't have Apple Dev accounts want to use it

**Comment by tmspzz** [comment]
yes, I don't have a distribution certificate. I agree with @kylejm , if possible please changed it to `iOS Developer`. 

This is the only framework among the many I use (Alamofire, KeyChainAccess, ResearchKit, Swiftz) that requires a distribution certificate.

I have opened an issue for this https://github.com/nickoneill/PermissionScope/issues/140

**Comment by ky1ejs** [comment]
@nickoneill @bre7 how's it looking for looking for the next release?

**Comment by nickoneill** [comment]
Sorry @kylejm, busy time of year. I'll work on this today and report back with what else needs to get done before we can get a release out the door.

**Comment by nickoneill** [comment]
I'm waiting to hear back on #125 with some updated changes. If that's resolved then we'll move ahead with 1.0.2 in the next day or so.

**Comment by ky1ejs** [comment]
Awesome, looking forward to it :blush:

**Comment by BryanRyczek** [comment]
hey guys I am new to Carthage and PermissionScope is my first Carthage install.

I am running Swift 2.1.1 and Xcode 7.2

In the top of my .swift file, I write 'import PermissionScope'  which autocompletes so I know the compiler is expecting its possible use. 

However, I get kicked this fatal error:

AnimationTestingViewController.swift:11:8: Module file was created by an older version of the compiler; rebuild 'PermissionScope' and try again: Carthage/Build/iOS/PermissionScope.framework/Modules/PermissionScope.swiftmodule/arm64.swiftmodule

Reading this thread, am I right to guess that I just have to wait for a new, supported version of PermissionScope to be released?

Thanks for any help

**Comment by nickoneill** [comment]
Hey @BryanRyczek, this is a known issue with PermissionScope right now (see #137), you can fix it immediately by using the `--no-use-binaries` option in Carthage. We'll fix it for our upcoming 1.0.2 release.

**Comment by nickoneill** [comment]
Should be set here with the proper 1.0.2 release.
